libraries/AP_GPS/AP_GPS_SBF.h

```diff
@@ -66,6 +66,18 @@ class AP_GPS_SBF : public AP_GPS_Backend
     "sem, PVT, 5\n",
     "spm, Rover, all\n",
     "sso, Stream2, Dsk1, postprocess+event+comment+ReceiverStatus, msec100\n"};
+    const char* _initialisation_blob_i[5] = {
```

Comment:
I tried this but for example if using the AsteRx-m2 (indeed you where right this one is single antenna. I had the AsteRx-m2a i mind for dual antenna mode) messages like INSNavGeod and AttEuler are not generated and rejected during the initialization. Therefore it might happen for the AsterRx-m2 not to work. Indeed setting multiple streams on the same port is well defined concept but if the message is not supported (either by hardware or the permission file) it fail to initialize. Unless you do it by your self directly in Rx Control or by sending custom commands over serial communication (that is how we did it in the past)

Reply:
Without the INSNavGeod and AttEuler messages the AsterRx-i S and the AsteRx-m2a will work. But its full capabilities will not be utilized. Like IMU corrected lat and lon or the heading, roll and pitch.